### ISSUE TITLE
Adding TensorBoard mesh breaks guild view

### ISSUE THREAD

**Issue by copaah** [guildai/guildai#178]
After adding a [TensorBoard mesh using Pytorch  ](https://pytorch.org/docs/stable/tensorboard.html#torch.utils.tensorboard.writer.SummaryWriter.add_mesh) `guild view` is broken:

```
$ guild view
ERROR: Error on request:
Traceback (most recent call last):
  File "/home/lenny/.local/lib/python3.6/site-packages/guild/view.py", line 350, in _runs_data
    run_id_prefix = req.args["run"]
  File "/home/lenny/.local/lib/python3.6/site-packages/werkzeug/datastructures.py", line 442, in __getitem__
    raise exceptions.BadRequestKeyError(key)
werkzeug.exceptions.BadRequestKeyError: 400 Bad Request: The browser (or proxy) sent a request that this server could not understand.During handling of the above exception, another exception occurred:Traceback (most recent call last):
  File "/home/lenny/.local/lib/python3.6/site-packages/werkzeug/serving.py", line 323, in run_wsgi
    execute(self.server.app)
  File "/home/lenny/.local/lib/python3.6/site-packages/werkzeug/serving.py", line 312, in execute
    application_iter = app(environ, start_response)
  File "/home/lenny/.local/lib/python3.6/site-packages/guild/serving_util.py", line 124, in app
    return dispatch(routes, env, start_resp)
  File "/home/lenny/.local/lib/python3.6/site-packages/guild/serving_util.py", line 113, in dispatch
    return handler(*args, **kw)(env, start_resp)
  File "/home/lenny/.local/lib/python3.6/site-packages/guild/view.py", line 344, in _handle_runs
    runs_data = _runs_data(req, data)
  File "/home/lenny/.local/lib/python3.6/site-packages/guild/view.py", line 352, in _runs_data
    return data.runs_data()
  File "/home/lenny/.local/lib/python3.6/site-packages/guild/commands/view_impl.py", line 69, in runs_data
    return list(self._runs_data_iter(self.runs()))
  File "/home/lenny/.local/lib/python3.6/site-packages/guild/commands/view_impl.py", line 88, in _runs_data_iter
    index = self._init_index(runs)
  File "/home/lenny/.local/lib/python3.6/site-packages/guild/commands/view_impl.py", line 101, in _init_index
    index.refresh(runs, ["scalar"])
  File "/home/lenny/.local/lib/python3.6/site-packages/guild/index2.py", line 385, in refresh
    self._scalar_reader.refresh(runs)
  File "/home/lenny/.local/lib/python3.6/site-packages/guild/index2.py", line 111, in refresh
    self._maybe_refresh_run_scalars(run, path, cur_digest, scalars)
  File "/home/lenny/.local/lib/python3.6/site-packages/guild/index2.py", line 123, in _maybe_refresh_run_scalars
    self._refresh_run_scalars(run, prefix, cur_digest, last_digest, scalars)
  File "/home/lenny/.local/lib/python3.6/site-packages/guild/index2.py", line 126, in _refresh_run_scalars
    summarized = self._summarize_scalars(scalars)
  File "/home/lenny/.local/lib/python3.6/site-packages/guild/index2.py", line 155, in _summarize_scalars
    for tag, val, step in scalars:
  File "/home/lenny/.local/lib/python3.6/site-packages/guild/tfevent.py", line 91, in __iter__
    [self._try_tfevent_v2, self._try_tfevent_v1], event, val
  File "/home/lenny/.local/lib/python3.6/site-packages/guild/util.py", line 63, in try_apply
    return f(*args)
  File "/home/lenny/.local/lib/python3.6/site-packages/guild/tfevent.py", line 107, in _try_tfevent_v2
    return val.tag, make_ndarray(val.tensor).item(), event.step
ValueError: can only convert an array of size 1 to a Python scalar
```

Using TensorBoard like:

```
tensorboard --logdir ~/.guild/runs/
```

Works without any issues.

**Versions**

```
$ python3 -m pip list | egrep '(guild|tensorboard|torch)'
guildai                       0.7.0rc9           
tensorboard                   2.2.1              
tensorboard-plugin-wit        1.6.0.post3        
torch                         1.4.0              
torchvision                   0.5.0 
```

**Comment by gar1t** [comment]
Weird - I must have closed this by accident!

**Comment by gar1t** [comment]
What a pain - GitHub automatically closes an issue when it sees "Fixes <issue>" in a commit message. It'd sure be nice to disable this unhelpful behavior but it looks like we have to take care when writing a commit message. How dumb.

We do NOT want to close issues via commit messages but rather through collaboration with users who take the time and effort to open an issue to help improve Guild! I apologize for the without-even-a-thank-you closing.

Thank you for this detailed report! :)

I created an issue doc this here:

https://github.com/guildai/issue-resolution/tree/master/178-adding-tensorboard-mesh-breaks-guild-view

These are used to track reproducibility of the issue and verify fixes when they're available.

This is fixed in master and will be available in the next release (rc10 is going out this week).

Let's keep this open until you get a chance to verify though.

Thanks again!

**Comment by copaah** [comment]
Sweet, I'll verify it when rc10 is out!

**Comment by gar1t** [comment]
Should be fixed in 0.7.0. If you're still seeing this behavior, please feel free to re-open. Thanks again!
